elements/lisk-dpos/src/delegates_list.ts

```diff
@@ -441,14 +465,14 @@ export class DelegatesList {
 			forgersListStr !== undefined
 				? (JSON.parse(forgersListStr) as ForgersList)
 				: [];
-		const delegatePublicKeys = forgersList.find(fl => fl.round === round)
+		const delegateAddresses = forgersList.find(fl => fl.round === round)
 			?.delegates;
 
-		if (!delegatePublicKeys) {
+		if (!delegateAddresses) {
 			throw new Error(`No delegate list found for round: ${round}`);
 		}
 
-		return shuffleDelegateListForRound(round, delegatePublicKeys);
+		return delegateAddresses;
```

Comment:
also here, why we are not shuffling delegate list as the function clearly states that?

Reply:
I was thinking to rename this function, but it is giving shuffled delegateList so i was hesitant to change it... 🤔

Reply:
In this case, then the function name is a bit misleading, it could be simply `getDelegateList`